docs/orchestration/faq/upgrade_environments.md

```diff
@@ -159,12 +159,12 @@ Agent](/orchestration/agents/fargate.md).
   flow.executor = DaskExecutor()
   ```
 
-- `ECSRun` (coupled with the [ECS Agent](/orchestration/agents/ecs.md))
+- `ECSRun` (coupled with the [ECS Agent](../agents/ecs.md))
   supports configuring ECS Tasks at a finer level than before. If you
   previously configured custom task definitions on the [Fargate
-  Agent](/orchestration/agents/fargate.md), you may be better served by
+  Agent](../agents/fargate.md), you may be better served by
   specifying these options via `ECSRun` objects instead. See the [ECSRun API
-  docs](/api/latest/run_configs.md#ecsrun) for a complete list of available
+  docs](/docs/api/0.15.13/run_configs.md#ecsrun) for a complete list of available
```

Comment:
The original link works fine in production and testing, and the change breaks the link. I'd revert this.